index.js

```diff
@@ -1,19 +1,20 @@
+// Starting State
```

Comment:
Your entire code is very well written and really easy to understand, especially with the comments you have left behind. 👍